common/workunit/wuattr.cpp

```diff
@@ -283,3 +305,16 @@ void setAttributeValueInt(IPropertyTree & tgt, WuAttr kind, __int64 value)
     else
         addGraphAttribute(&tgt, info.childPath)->setPropInt64("@value", value);
 }
+
+bool isListAttribute(WuAttr kind)
+{
+    const WuAttrInfo & info = attrInfo[kind-WaNone];
+    return (info.multiKind && info.multiKind == kind);
+}
+
+bool getListAttribute(WuAttr kind)
+{
+    const WuAttrInfo & info = attrInfo[kind-WaNone];
+    WuAttr multiKind = info.multiKind;
+    return (multiKind && multiKind != kind) ? multiKind : WaNone;
```

Comment:
Again,the first part of this condition (multikind) should be info.multiKind != WaNone